x-pack/plugins/session_view/public/components/ProcessTreeNode/index.tsx

```diff
@@ -85,7 +88,15 @@ export function ProcessTreeNode({
   const { interactive } = processDetails.process;
 
   const renderChildren = () => {
-    const { children } = process;
+    let { children } = process;
+
+    // we pass an array of orphans to the session leader 
+    // for lack of a better approach, we just mix the orphans with its children and re-sort by timestamp.
+    // we could just add orphans to the children of the session leader in useProcessTree, but
+    // it makes it difficult to re-parent them when their parent actually shows up (e.g in the case of reverse pagination)
+    if (orphans) {
+      children = [...children, ...orphans].sort(sortProcesses);
```

Comment:
this actually doesn't do anything right now, as EventConverter is not setting process.start right now.. Will address this in the ticket to update field names to match ECS.